src/ResourceManager/Reservations/Commands.Reservations.Test/ScenarioTests/ReservationTests.ps1

```diff
@@ -0,0 +1,205 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+$subscriptionId ="98df3792-7962-4f18-8be2-d5576f122de3"
```

Comment:
You should not hard code these values.  contributors will need to be able to run these tests live.

Reply:
same as above comment, currently we can only create reservations through web portal and certain fields need to be hardcoded in this test.

Reply:
@corquiri if thsi is the case, then you need to provide instructions in test comments on how to set up the tests.  You can file an issue for this and link it here if there is not time for this today.

Reply:
https://github.com/Azure/azure-powershell/issues/4946